.github/workflows/ci_tests.yaml

```diff
@@ -41,20 +41,24 @@ jobs:
             isDraft: true
           - os: windows-latest
             isDraft: true
-          - os: ubuntu-latest
-            python-version: 3.7
-            isDraft: true
-          - os: ubuntu-latest
-            python-version: 3.8
-            isDraft: true
+          # - os: ubuntu-latest
+          #   python-version: 3.7
+          #   isDraft: true
+        # Pair Python 3.7 with NumPy 1.17 and Python 3.9 with NumPy 1.20
+        include:
+          - python-version: 3.7
+            numpy-version: '1.17'
+          - python-version: 3.9
+            numpy-version: '1.20'
```

Comment:
The NumPy versions are paired with Python versions here using 'environment variables' according to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-environment-variables-in-a-matrix.